api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -560,6 +562,7 @@ function treasuryProposalReducer(
 ) {
   return {
     id: `treasury-`.concat(proposal.index || proposal.votes.index),
+    proposalId: proposal.index || proposal.votes.index,
```

Comment:
why are there 2 possibilities here?

Reply:
because treasury proposals can have two different formats: council or top. This has given me too some headaches